index.html

```diff
@@ -270,6 +268,110 @@ <h4>
             or bypassing the UI if there is only a single share target.
           </div>
         </section>
+        <section>
+          <h3>
+            `canShare(data)` method
```

Comment:
Should this be marked as historical if we think this is a broken approach?

Reply:
Only if we are not going to ship an alternative - but I don't think anyone plans to.

Reply:
> Only if we are not going to ship an alternative

Are going to?

Reply:
Oops, yes... there is no plans for any engine to ship anything different. Let's just ship this and be done for now. If we need a new method, we can add it when there is demand for it.

Reply:
I don't think Gecko would want to ship anything proven to be broken as a sole solution 🤔

cc @annevk, context: https://github.com/w3c/web-share/issues/108#issuecomment-697689527

Reply:
That's ok, but if we can't shift the other implementers then we need to make a process call here (i.e., Mozilla can "formally object" and we figure it out from there). 

We have two engines that already ship `.canShare()` already (and have now for over a year).

Reply:
I think it depends on real world usage of canShare.

Reply:
You are right, that too... however, we (you and I) are kinda limited in that we can't actually get that data. 

We are at an impasse here :( without input from other implementers willing to change (or even respond to issues), I don't know what we can do 😭

Reply:
> You are right, that too... however, we (you and I) are kinda limited in that we can't actually get that data.

https://chromestatus.com/metrics/feature/timeline/popularity/2737